docs/access/jupyterlab.md

```diff
@@ -0,0 +1,238 @@
+
+When resources are granted the page redirects to the JupyterLab session, where you can browse, open and execute notebooks on the compute nodes. A new notebook with a Python 3 kernel can be created with the menu `new` and then `Python 3` . Under `new` it is also possible to create new text files and folders, as well as to open a terminal session on the allocated compute node.
+
+!!! tip "Debugging"
+    The log file of a JupyterLab server session is saved on `$HOME` in a file named `slurm-<jobid>.out`. If you encounter problems with your JupyterLab session, the contents of this file can be a good first clue to debug the issue.
+
+??? warning "Unexpected error while saving file: disk I/O error."
+    This error message indicates that you have run out of disk quota.
+    You can check your quota using the command `quota`.
+
+
+[](){#ref-jupyter-runtime-environment}
+## Runtime environment
+
+A Jupyter session can be started with either a [uenv][ref-uenv] or a [container][ref-container-engine] as a base image. The JupyterHub Spawner form provides a set of default images such as the [prgenv-gnu][ref-uenv-prgenv-gnu] uenv or the [NGC Pytorch container][ref-software-ml] to choose from in a dropdown menu. When using uenv, the software stack will be mounted at `/user-environment`, and the specified view will be activated. For a container, the Jupyter session will launch inside the container filesystem with only a select set of paths mounted from the host. Once you have found a suitable option, you can start the session with `Launch JupyterLab`.
+
+??? info "Using remote uenv for the first time."
+    If the uenv is not present in the local repository, it will be automatically fetched.
+    As a result, JupyterLab may take slightly longer than usual to start.
+
+!!! warning "Ending your interactive session and logging out"
+    The Jupyter servers can be shut down through the Hub. To end a JupyterLab session, please select `Hub Control Panel` under the `File` menu and then `Stop My Server`. By contrast, clicking `Logout` will log you out of the server, but the server will continue to run until the Slurm job reaches its maximum wall time.
+
+If the default base images do not meet your requirements, you can specify a custom environment instead. For this purpose, you supply either a custom uenv image/view or CE TOML file under the section `Advanced options` before launching the session. The supported uenvs are compatible with the Jupyter service out of the box, whereas container images typically require the installation of some additional packages. 
```

Comment:
```suggestion
If the default base images do not meet your requirements, you can specify a custom environment instead. For this purpose, you supply either a custom uenv image/view or [container engine (CE][ref-container-engine] TOML file under the section `Advanced options` before launching the session. The supported uenvs are compatible with the Jupyter service out of the box, whereas container images typically require the installation of some additional packages. 
```
This is the first time CE is mentioned, so I'd use the full name and a link (even if the same is already used for "container").